tests/src/JIT/Performance/CodeQuality/Math/Functions/Abs.cs

```diff
@@ -0,0 +1,81 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Xunit.Performance;
+
+namespace Functions
+{
+    public static partial class MathTests
+    {
+        private const double absDoubleDelta = 0.0004;
+        private const double absDoubleEpsilon = 2.22e-16;
```

Comment:
If this epsilon value is same across all tests (looks like it is...), I'd suggest you just have one definition for it.

Also you should add a comment on why it has the value it does.